nomad/resource_job_test.go

```diff
@@ -305,22 +367,40 @@ func testResourceJob_updateCheck(s *terraform.State) error {
 	return nil
 }
 
-var testResourceJob_parameterizedJob = `
+func TestResourceJob_vault(t *testing.T) {
+	r.Test(t, r.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []r.TestStep{
+			{
+				Config: testResourceJob_vaultConfig,
+				Check:  testResourceJob_initialCheck,
+			},
+		},
+		CheckDestroy: testResourceJob_checkDestroy("test"),
+	})
+}
+
+var testResourceJob_vaultConfig = `
+provider "nomad" {
+	vault_token = "terraform-provider-nomad-token"
```

Comment:
This (obviously) isn't a valid token. Is the intent here to get a working token passed in? Is there any reason not to use the environment variable configuration option?